todo/src/App.js

```diff
@@ -0,0 +1,49 @@
+import React, { Component } from 'react'; 
+import './App.css';
+import {connect} from 'react-redux';
+import TodoForm from './Components/TodoForm'
+
+
+// Your application should have an input field, a submit button,
+// and a list of items that represents your todo list.
+
+class App extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: []
```

Comment:
Remember with this redux app, the todos state is being taken care of via redux.  You don't necessarily have to keep track of component state inside of todos.